src/theme/playpen_editor/editor.js

```diff
@@ -2,10 +2,10 @@ window.editors = [];
 (function(editors) {
     if (typeof(ace) === 'undefined' || !ace) {
         return;
-    }
-
-    $(".editable").each(function() {
-        let editor = ace.edit(this);
+    
+    
+    Arom.from(document.querySelectorAll('.editable')).forEach(function(editable) {
```

Comment:
What is `Arom`? Is this supposed to be `Array`?

Reply:
Yes, fixed 👍